addons/website/views/res_config_settings_views.xml

```diff
@@ -133,14 +133,6 @@
                                 <button type="object" name="action_open_robots" string="Edit robots.txt" class="btn-link" icon="fa-android" noSaveDialog="true"/>
                             </div>
                         </setting>
-                        <setting id="robots_setting" string="Sitemap" help="Sitemap.xml: Help search engine crawlers to find out what pages are present and which have recently changed, and to crawl your site accordingly. This file is automatically generated by Odoo.">
```

Comment:
Nice, we get rid of this crappy `id="robots_setting"` copy paste from a few lines above at the same time :+1: :+1: